File: elixir/test/solutions/0242_valid_anagram/valid_anagram_test.exs
defmodule LeetCode.Solutions.ValidAnagramTest do
  use ExUnit.Case, async: true

  alias LeetCode.Solutions.ValidAnagram

  test "Case 1 works" do
    assert ValidAnagram.call("anagram", "nagaram") == true
  end

  test "Case 2 works" do
    assert ValidAnagram.call("rat", "car") == false
  end
end

File: ruby/test/solutions/0242_valid_anagram/valid_anagram_test.rb
# frozen_string_literal: true

require 'test_helper'
require_relative '../../../lib/solutions/0242_valid_anagram/valid_anagram'

class ValidAnagramTest < Minitest::Test
  def test_case_1_works
    assert anagram?('anagram', 'nagaram')
  end

  def test_case_2_works
    refute anagram?('rat', 'car')
  end
end
